readme.md

```diff
@@ -60,6 +61,10 @@ Apps that help you manage your extensions.
 - [@wext/shipit](https://github.com/LinusU/wext-shipit) - Tool to automatically publish to Chrome Web Store, Mozilla Addons and Opera Addons.
 - [wext-manifest-loader](https://github.com/abhijithvijayan/wext-manifest-loader) - Webpack loader that lets you specify `manifest.json` properties to appear only in specific browsers.
 - [webextension-manifest-loader](https://github.com/jsmnbom/webextension-manifest-loader) - Webpack loader that loads browser tailored manifest.json. It also imports all importable properties, allowing you to have 'manifest.json' as your only webpack entry point.
+- [webextension-feedback-popup](https://github.com/nikoisonfire/webext-feedback-popup) - Customizable Popup, that asks a user to leave a rating after a period of using the extension (e.g. 7 days after install). Works for FF, Chrome, Edge and Safari.
```

Comment:
Most definitely not

Reply:
Why not? Not compliant with TOS?

Reply:
I get that we need/like ratings, but popups are not the way to go, nobody likes popups.

Reply:
How else would you go about it? Grammarly and every other big extension does it in the same way.

How many of the average chrome/ff users ever do drop a rating if not asked for it?

FYI by default it only shows once and can be easily clicked away. I think it's very non-invasive.

Reply:
> Grammarly and every other big extension does it in the same way.

And they're all _not awesome._ Any extension that injects popups into a page gets uninstalled and rated negatively. A while ago a famous extension added a very similar screen and the guy removed the GitHub repo to stop the influx of comments about this, if I remember correctly. I can't find it anymore. Here's a negative review mentioning it: https://addons.mozilla.org/en-US/firefox/addon/i-dont-care-about-cookies/reviews/1380210/

> it only shows once and can be easily clicked away

The same can be said by any popups, but they're still popups. The module also provides a way to show it multiple times so that's not entirely true

https://github.com/nikoisonfire/webext-feedback-popup/blob/dc74fda0fe9d6e32a773180e6ebff42af7659964/src/index.ts#L33

> How else would you go about it?

In the options page or any extension page, as a regular link/text. Example (not ratings but similarly static "advertising" text):

<img width="671" alt="Screen Shot 5" src="https://user-images.githubusercontent.com/1402241/124931037-0c088580-e02c-11eb-8860-2ecfeb412cf3.png">

Reply:
I can agree with some of your points, some not. I guess the one about the review is annoying, some developers overdo it. Then again people rate restaurants only if the food was bad. Nobody leaves a 5* everywhere they go.

Do you remember when you started developing extensions? What was that feeling like when you had few users and no ratings? Did you think there was something wrong with your? Wouldn't you want to know?

Web users are so f* entitled to free stuff nowadays that asking for feedback is apparently like asking for an organ. 

I guess this discussion is going nowhere. Fine, leave it out.